### ISSUE TITLE
Using NISQA as a loss function

### ISSUE THREAD

**Issue by akashrajkn** [gabrielmittag/NISQA#27]
Hi, thank you very much for the work, the repo and the pretrained models.

Can we use NISQA trained weights (`nisqa_tts`) as a loss function to train a TTS/audio synthesis model?

Thanks in advance

**Comment by gabrielmittag** [comment]
Hi, I have not tried that but it sounds interesting and would be worth investigating. I am not sure if the model's outputs are accurate enough for doing that, especially when it's applied to completely new systems, so you might need to retrain the model with your own data. Let me know if you have more questions about that.

**Comment by stonelazy** [comment]
@akashrajkn Any luck with this interesting experiment ?
